styles/normalize.css

```diff
@@ -0,0 +1,427 @@
+/*! normalize.css v3.0.2 | MIT License | git.io/normalize */
```

Comment:
since you won't be changing the code, consider adding normalize to your gitignore